com.sleepfiles.OSCAR.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
you install the one provided from upstream you don't need this

Reply:
Have to send a modified one for now, but I'll commit it upstream once it's okay.

Reply:
in that case we patch, or use `desktop-file-edit` in the post install.

Still you don't use that file as you install the one from the package.

Reply:
This file should actually be removed now.

Reply:
This file should actually be removed now.